client/blocks/color-scheme-picker/docs/example.jsx

```diff
@@ -0,0 +1,43 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Notice from 'components/notice';
+import ColorSchemePicker from 'blocks/color-scheme-picker';
+
+class ColorSchemePickerExample extends PureComponent {
+	static displayName = 'ColorSchemePicker';
+
+	state = {
+		selectedColorScheme: null,
+	};
+
+	handleColorSchemeSelection = colorScheme => {
+		this.setState( { selectedColorScheme: colorScheme } );
+	};
+
+	render() {
+		return (
+			<Card>
+				<ColorSchemePicker
+					temporarySelection={ true }
+					onSelection={ this.handleColorSchemeSelection }
+				/>
+
+				{ this.state.selectedColorScheme &&
+					<Notice
+						text={ `Selected color scheme: ${ this.state.selectedColorScheme }` }
```

Comment:
We should use the [localize HoC](https://github.com/Automattic/i18n-calypso) and the `translate` prop it provides for this string. 
I think it may end up looking a little like this:
```js
...
text={ 
  translate( 'Selected color scheme: %s', {
    args: this.state.selectedColorScheme
  } )
}
```
I'm not sure if this will open up those values to translation, but if done right this could help us avoid using `translate()` in the constants file.

( More details on [substituting strings](https://github.com/Automattic/i18n-calypso#string-substitution) in `localize`)